lib/kredis/types/proxy/failsafe.rb

```diff
@@ -1,26 +1,33 @@
 module Kredis::Types::Proxy::Failsafe
-  def initialize(*)
-    super
-    @fail_safe_suppressed = false
-  end
+  extend ActiveSupport::Concern
 
-  def failsafe
-    yield
-  rescue Redis::BaseError
-    raise if fail_safe_suppressed?
+  included do
+    mattr_accessor :fail_safe_enabled, default: true
```

Comment:
This attr can live on Kredis::Types::Proxy::Failsafe itself without mixing in to the including class and, in this context, it can be shorted to `:enabled`

Reply:
Good suggestion. I also changed fail_safe_suppressed -> suppressed.